ckanext/tabledesigner/templates/base.html

```diff
@@ -0,0 +1,7 @@
+{% ckan_extends %}
+
+{% block styles %}
+  {{ super() }}
+  {% asset 'ckanext-tabledesigner/fields' %}
+  {% asset 'ckanext-tabledesigner/datatables_buttons' %}
```

Comment:
Can we load these just in the templates they are needed rather than the base one? Otherwise they get loaded in all pages. Maybe `ckanext/tabledesigner/templates/datastore/dictionary.html` and `ckanext/tabledesigner/templates/package/resource_read.html`? That seem to work fine for the CSS asset

Reply:
For sure. I was following recommendation from the docs https://docs.ckan.org/en/2.10/theming/webassets.html#adding-css-and-javascript-files-using-webassets (assuming including all js in the base means the bundle of JS will be cached better?) but it makes more sense to me to only include them in the pages that need them.